lib/oxidized/model/riverbed.rb

```diff
@@ -0,0 +1,131 @@
+class Riverbed < Oxidized::Model
+  using Refinements
+
+  # Define the prompt
+  PROMPT = /^.*\s*[\w-]+\s*[#>]\s*$/i
+  prompt PROMPT
+
+  # Define comment character
+  comment '! '
+
+  # Use procs to remove unwanted lines
```

Comment:
- The "Last login" message should never appear, as Oxidized does not interpret the initial prompt.
- Do not use procs directly. Use [cmd :all](https://github.com/ytti/oxidized/blob/master/docs/Ruby-API.md#cmd)

Reply:
Removed the entire procs block.